recipes/pykdtree/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.1.1" %}
+
+package:
+    name: pykdtree
+    version: {{ version }}
+
+source:
+    fn: pykdtree-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/p/pykdtree/pykdtree-{{ version }}.tar.gz
+    md5: 936b0844b3d06d8f2f2a1ca40f587cfb
+    patches:
+        - win_comp.patch  # [win]
```

Comment:
Giving that a go. Let's see what happens. Thanks. (I am turning in :zzz: we'll see the results tomorrow.)